core/os/os.h

```diff
@@ -95,6 +97,77 @@ class OS {
 		RENDER_SEPARATE_THREAD
 	};
 
+	// Feature system - char is used so we can have 64 features at max.
+	enum Feature {
+		ANDROID,
+		MOBILE_SYSTEM_FONTS,
```

Comment:
`MOBILE_SYSTEM_FONTS` and `SYSTEM_FONTS?`. Why there are two features for the same thing?

Reply:
in some platforms system_fonts can be calculated at compile time, others not, but still in gdscript you just do:

```gd
~if system_fonts

~endif
```

its called mobile, because in mobile its at compile time, but just discovered that in Macos is at compile time too, which is weird!